Ariel/Ariel/Controllers/CityViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CityViewController.swift
+//  Ariel
+//
+//  Created by Alina Zaitseva on 5/17/18.
+//  Copyright © 2018 Alina Zaitseva. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+import MapKit
+
+class CityViewController: UIViewController, UISearchBarDelegate, CLLocationManagerDelegate {
+
+    @IBOutlet weak var searchBar: UISearchBar!
+    let locationManager = CLLocationManager()
+    var cityManager = CityModel()
+    var weatherManager = WeatherViewController()
+    var delegate: PageViewController?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        searchBar.delegate = self
+        searchBar.resignFirstResponder()
+    }
+    
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        searchBar.resignFirstResponder()
+        if let locationString = searchBar.text, !locationString.isEmpty {
+            guard let connection = delegate else { return }
+            connection.cityManager.addCity(locationString)
+            connection.orderedController = connection.addPage()
```

Comment:
I think that `addPage()` and `setViewControllers` should be called from the inside of `addCity` method by connection(?) or delegate(?)